src/expertsystem/reaction/__init__.py

```diff
@@ -429,6 +453,184 @@ def _solve(
         return solver.find_solutions(*state_graph_node_settings_pair)
 
 
+def check_reaction_violations(
+    initial_state: Union[StateDefinition, Sequence[StateDefinition]],
+    final_state: Sequence[StateDefinition],
+) -> Set[Tuple[str, ...]]:
+    """Determine violated interaction rules for a given particle reaction.
+
+    .. warning:: This function does only guarantees to find P, C and G parity
+      violations, if its a two body decay. If all initial and final states have
```

Comment:
it's

Reply:
44e8ea04bba93a7a35021d254c48a9afa1d045b3